teams/deployment/membership.json

```diff
@@ -0,0 +1,44 @@
+{
+    "_comment": [
+        "Items of lists in this file are in alphabetical order. Please keep it on modification."
+    ],
+    
+    "maintainers": [
+        "DanielZhangQD",
+        "Yisaer",
```

Comment:
I think `committers` is more proper for me instead of `maintainer`.

Reply:
As you wish, updating...

Reply:
Updated.